instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_instrumentation.py

```diff
@@ -134,10 +127,12 @@ def test_s3_client(self):
         assert spans
         span = spans[2]
         self.assertEqual(
-            span.resource,
-            Resource(
-                attributes={"endpoint": "s3", "operation": "listobjects"}
-            ),
+            span.attributes,
+            {
+                "aws.operation": "ListObjects",
+                "aws.region": "us-west-2",
```

Comment:
Any reason on why don't we validate status code?

Reply:
Because this is a `ParamValidationError` the `botocore` library will never send the HTTP request and so there's no `http.status_code` to check!

But that's a good point! I can add a check to make sure this span was recorded as an error.

```
self.assertIs(
    span.status.status_code, trace_api.status.StatusCode.ERROR,
)
```